lispify/lispify.py

```diff
@@ -9,11 +9,18 @@
 so that the lispify method ignores it.
 """
 
+from __future__ import absolute_import
+
 from numbers import Number
+from sys import version_info
 import warnings
 
-from util import subclasses, camel_case_to_lisp_name
+from lispify.util import subclasses, camel_case_to_lisp_name
+
 
+# python 3 compatibility functions
+def string_types():
```

Comment:
Have you considered http://python-future.org/compatible_idioms.html#basestring ?

My thoughts:
   * **Advantage:** third party library, robust, a layer of abstraction that we don't have to worry about or maintain
   * **Disadvantage:** additional dependency (although probably a light one)

Your thoughts?